downstream/modules/platform/ref-scaling-control-nodes.adoc

```diff
@@ -0,0 +1,15 @@
+[id="ref-scaling-control-nodes"]
+
+= Benefits of scaling control nodes
+
+Control and hybrid nodes provide control capacity. They provide the ability to start jobs and process their output into the database. Every job is assigned a control node. In the default configuration, each job requires 1 capacity unit to control. For example, a control node with 100 capacity units can control a maximum of 100 jobs.
+
+Vertically scaling a control node by deploying a larger virtual machine with more resources increases the following capabilities of the control plane:
+* The number of jobs that a control node can perform control tasks for, which requires both more CPU and memory
+* The number of job events a control node can process concurrently
+
+Scaling CPU and memory in the same proportion is recommended (e.g. 1 CPU: 4 GB RAM). Even when memory consumption is high, increasing the CPU of an instance can often relieve pressure. The majority of the memory that control nodes consume is from unprocessed events that are stored in a memory-based queue. 
```

Comment:
I would avoid latin abbreviations as per SG, suggestion "...is recommended, for example, 1 CPU: 4 GB RAM."